Objects/StrategyCardAutomator.ttslua

```diff
@@ -968,27 +968,26 @@ function _addTechnologyPrimarySecondary(layout)
 
 	if not _setupHelper.getPoK() then
 		-- remove pok tech from ui
-
+		local _techSourceSet = _technologyHelper.getTechSourceSet()
```

Comment:
Opted to get list of tech sources instead of single cross-object calls for each tech.

Reply:
Nice.  I'm less worried about one-time setup stuff being slow vs happens-during-runtime but looks good!